[READY] Map fixer

Vasily2013 opened the pull request:
## About The Pull Request

fixes #401 and moves some air alarms in xenobio so they're on turfs that always have atmos passable. Also fixes #436 and fixes #460 

## Why It's Good For The Game

bugs are bad.

## Changelog
:cl:
add: New air alarm in xenobio.
Tweak: Moves air alarm previously under xenobio airlock door to the airlock proper.
Tweak: conveyor in atmos can now (in theory) go both ways
Fix: Now gives XO on Hammerhead and Vago the TGUI based id modification consoles.
Fix: Bridge windows now match NSV13 theme.
/:cl:

Cenrus commented:
There's a missing cable under the door leading to the service break room, I deleted it on accident, and it prevents the kitchen APC from recieving power.

Vasily2013 commented:
> On the NSV Hammerhead
> 
> There's a missing cable under the door leading to the service break room, I deleted it on accident, and it prevents the kitchen APC from recieving power.

Fixed, and also patched a missing cable in the courtroom that I noticed last time I played engi.
